.github/workflows/release_staging.yaml

```diff
@@ -69,7 +69,7 @@ jobs:
     - name: Upload coverage reports to Codecov with GitHub Action
       uses: codecov/codecov-action@v3
       with:
-        files: ./coverage1.txt,./coverage2.txt,./coverage3.txt,./addons/coverage.txt,./pinniped-components/post-deploy/coverage.txt,./pinniped-components/tanzu-auth-controller-manager/coverage.txt,./cli/core/coverage.txt,./cli/runtime/coverage.txt,./tkg/coverage.txt,./featuregates/client/cover.out,./featuregates/controller/cover.out,./capabilities/client/cover.out,./capabilities/controller/cover.out
+        files: ./coverage1.txt,./coverage2.txt,./coverage3.txt,./addons/coverage.txt,./pinniped-components/post-deploy/coverage.txt,./pinniped-components/tanzu-auth-controller-manager/coverage.txt,./cli/core/coverage.txt,./cli/runtime/coverage.txt,./tkg/coverage.txt,./featuregates/client/cover.out,./featuregates/controller/cover.out,./capabilities/client/cover.out,./capabilities/controller/cover.out./packages/cluster-api-provider-azure/test/coverage2.txt,./packages/cluster-api-provider-vsphere/test/coverage2.txt,./packages/cluster-api/test/coverage2.txt,./packages/cluster-api-provider-docker/test/coverage2.txt,./packages/cluster-api-bootstrap-kubeadm/test/coverage2.txt,./packages/cluster-api-provider-aws/test/coverage2.txt,
```

Comment:
Here too, the command seems to be missing.

Reply:
Will get that changed.